README.md

```diff
@@ -14,8 +14,57 @@ the packages individually, for example:
     make -C debs/xenial/archivematica
     make -C debs/xenial/archivematica-storage-service
 
-### Test packages
+#### Using parameters
 
+Most makefiles support parameters in the form of environment variables. They 
+may be a bit different between packages, but the most common are ``BRANCH``,
+``VERSION`` and ``RELEASE``.
+
+So in order to build a specific branch / version, this command can be used from
+the folder of the package we want to build:
+
+    make BRANCH=qa/1.x VERSION=1.7.0 RELEASE=rc5
+
+Keep in mind that the Makefiles are a bit recursive, they will invoke docker, 
```

Comment:
In L33 you wrote "Some makefiles", here reads "the Makefiles". Not sure which one is correct but I suggest to pick one and be consistent.